python/SMHiggsBuilder.py

```
@@ -303,7 +309,7 @@ def makePartialWidthUncertainties(self):
                         if D in DS:
                             var = self.modelBuilder.out.var("HiggsDecayWidthTHU_%s" % K2)
                             break
-                    if var is None:
+                    if var == None:
```

Ruff (0.14.3)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)